docs/manual/unity/helloworld-unity-importwebrtc.md

```diff
@@ -15,7 +15,7 @@ The Unity library is distributed as a package containing all those components, i
 > [!NOTE]
 > If an existing Unity project manifest already contains a `Microsoft Mixed Reality` entry in the `scopedRegistries` section, is is recommended that it be removed.
 
-In the following, we describe the latter method, which essentially consists in manually replicating the steps that the Mixed Reality Feature Tool would oherwise perform automatically for you.
+In the following, we describe the latter method, which essentially consists of manually replicating the steps that the Mixed Reality Feature Tool would otherwise perform automatically for you.
```

Comment:
I think "consists in" is more correct here (see https://english.stackexchange.com/questions/61600/consist-in-vs-consist-of)
```suggestion
In the following, we describe the latter method, which essentially consists in manually replicating the steps that the Mixed Reality Feature Tool would otherwise perform automatically for you.
```

Reply:
Fascinating, I'm not a native english speaker, I've never actually heard of the preposition "in" being used in conjunction with "consists"! Cool, I'm satisfied.